contracts/TxPermission.sol

```diff
@@ -79,26 +78,34 @@ contract TxPermission is OwnedEternalStorage {
     function allowedTxTypes(
         address _sender,
         address _to,
-        uint256 /*_value*/, // solhint-disable-line space-after-comma
+        uint256 _value,
         uint256 _gasPrice
     )
         public
         view
         returns(uint32, bool)
     {
         if (_gasPrice > 0 || isSenderAllowed(_sender)) {
-            // Let `_sender` create any transactions with non-zero gas price
-            // or if he is in the allowedSenders list
+            // Let `_sender` create any transactions if any of the following are true:
+            // * the gas price is non-zero
+            // * they are in the allowedSenders list
+            // * they are the owner
```

Comment:
Do we really want to allow all these transactions?
Maybe that's for a separate PR, but in the long term I think we should restrict these as much as possible and allow only the service transactions we expect.
Do we want fixed "allowed senders" in this contract at all? And should the owner of this contract really be allowed to do anything at zero cost?

Reply:
> Do we want fixed "allowed senders" in this contract at all?

They are not fixed: allowed senders can be added or removed by an owner of `TxPermission` contract when needed.

> And should the owner of this contract really be allowed to do anything at zero cost?

Owner of `TxPermission` is not allowed to use zero cost by default unless he is whitelisted with `addAllowedSender()` function. The `// * they are the owner` line should be removed.

Reply:
> can be added or removed

You're right, I shouldn't have said "fixed", but still: Do we want that functionality at all, where you can configure through contract calls an array of allowed senders? Shouldn't we just only allow that for the validators and nobody else?

Reply:
Yes, we need an ability to add/ remove whitelisted accounts for deploying bridge's contracts after the network starts.